Libraries/test/TestServerlessApp.IntegrationTests/DeploymentScript.ps1

```diff
@@ -0,0 +1,38 @@
+$ErrorActionPreference = 'Stop'
+try
+{
+    Push-Location $PSScriptRoot
+    $guid = New-Guid
+    $suffix = $guid.ToString().Split('-') | Select-Object -First 1
+    $identifier = "test-serverless-app-" + $suffix
+    cd ..\TestServerlessApp
+
+    # Replace bucket name in aws-lambda-tools-defaults.json
```

Comment:
Set the `--s3-bucket` parameter when calling `deploy lambda deploy-serverless` instead of changing the aws-lambda-defaults.json file.

Reply:
I need the stackName and the s3-bucketName in the actual tests. That is why I write them in the `aws-lambda-defaults.json file`. I tried setting environment variables to avoid modifying the `aws-lambda-defaults.json file` but unix based OS do not support setting user or machine specific environment variables. They only support setting Process based variables.